src/main/java/com/github/havardh/javaflow/phases/parser/java/TypeFactory.java

```diff
@@ -117,7 +117,7 @@ private static String extractKeyType(String typeLiteral) {
 
   private static String extractValueType(String typeLiteral) {
     int index = typeLiteral.indexOf(",");
-    return typeLiteral.substring(index+2, typeLiteral.length() - 1);
+    return typeLiteral.substring(index + 1, typeLiteral.length() - 1);
```

Comment:
Is this a removed whitespace character?

Reply:
Yes, I think they changed the format from `Map<String, String>` to `Map<String,String>`. But I realize now that it might have been a bit lazy on my side: Maybe we should better use something like `trim()`, right?